docs/adr/0028-a-feedback-arc-and-its-quote-are-one-mechanism.md

```
@@ -0,0 +1,546 @@
+    verify: { command: 'test "$(cat /tmp/t090/out-a.txt)" = "alpha"' }
+  feedback: { rerun: qa-a/build, max: 1 }
+```
+
+The loop ran **exactly as designed**. `qa-a/check` failed its verify, the
+engine wrote `feedback/qa-a~check.out`, and re-ran `qa-a/build`. And nothing
+was repaired, because `qa-a/build`'s prompt never contains
+`{{ feedback.qa-a/check }}`. The prompt *branches on feedback that can never
+arrive*: no FEEDBACK section ever appears below, so the model wrote `draft` a
+second time and the second round failed identically to the first.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)